### ISSUE TITLE
Stream connector tutorial - for dynamic balances?

### ISSUE THREAD

**Issue by RSchiemann** [justnielsen/ModelicaTutorials#1]
Hi,

thanks a lot for your very understandable tutorials concerning correct usage of Modelica stream operators. They greatly helped me to use them correctly.

In a next step, I would like to understand how to write dynamic energy balances in a correct, robust and elegant way. Could you please give advice or - even better - add a small section to your tutorials?

Best regards,
Robert

**Comment by justnielsen** [comment]
Hi Robert,

I'm glad that you found it useful. I've just committed a very simple dynamic pipe model. A few comments:

- The fluid properties are still very simple and in metric units. This makes some of the equations a bit clumsy because of unit conversions. When, at some point, you choose use `Modelica.Media` all these unit conversions will be unnecessary and the model can just look up the needed properties (internal energy etc.)
- I have only included the 'thermal' dynamic effects since I have assumed the fluid to be incompressible. Again, when you switch to `Modelica.Media` this will be handled in the medium model.
- I have only implemented a steady-state initialization
